src/alert/alert.scss

```
@@ -17,43 +18,52 @@
     background-color: t(iui-color-background-1);
   }
```

```suggestion
  }
  @media (forced-colors: active) {
    background-color: Canvas;
  }
```
Adds `forced-color` support.